probe-rs-tools/src/bin/probe-rs/util/cli.rs

```diff
@@ -645,8 +645,19 @@ where
     I: Future,
 {
     let mut run = std::pin::pin!(f);
+    #[cfg(unix)]
+    let terminate = async {
+        tokio::signal::unix::signal(tokio::signal::unix::SignalKind::terminate())
+            .expect("failed to install signal handler")
```

Comment:
I do wonder if we should fall back to pending in this case, instead of panicking. What do you think?

Reply:
Maybe
```
    #[cfg(unix)]
    let terminate = async {
        tokio::signal::unix::signal(tokio::signal::unix::SignalKind::terminate())?
            .recv()
            .await;
        std::io::Result::Ok(())
    };
```
to match what `tokio::signal::ctrl_c()` does for SIGINT?

Reply:
I think it's reasonable to panic here, especially since the current behavior for Ctrl-C is to resolve with an error and exit immediately if it fails to register SIGINT. Panicking provides a clearer signal that something went wrong and why the process is exiting.